docs/Introduction.md

```diff
@@ -87,6 +87,8 @@ app.start();
 There is also a helper command available to generate a small example application for you.
 
 ```
+$ mkdir myapp && cd myapp
+$ npm i @mojojs/core
```

Comment:
I think we already discussed this in the other PR. 🙄

Reply:
Ok, personally *I* think repetition and full examples are better in tutorial guides, while reference guides should be concise and DRY. But I'll update to not include the full example since you hate it so much.

Reply:
Oh wait. I didn't even consider I'd included this change in the intro in this branch. Sorry. 🤦🏻